frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -1044,14 +1044,7 @@ mod tests {
 			roll_to(25);
 			assert!(MultiPhase::current_phase().is_unsigned());
 
-			assert_eq!(
-				MultiPhase::mine_check_save_submit().unwrap_err(),
-				MinerError::PreDispatchChecksFailed(DispatchError::Module(ModuleError {
-					index: 2,
-					error: 1,
-					message: Some("PreDispatchWrongWinnerCount"),
-				})),
-			);
+			assert_ok!(MultiPhase::mine_check_save_submit());
```

Comment:
This test should still trigger the same error `PreDispatchWrongWinnerCount`.

You have two ways to do this: 

1. alter the solution manually to have too few winners.
2. alter the solution manually to have too many winners.

Reply:
Done, if you can please let me know if this is what you had in mind?